### PR TITLE
Update send.md

### PR DESCRIPTION
**Pull request by paulosacramento**
Changes based on the recommendations made by @danielnordh here: https://github.com/BitcoinDesign/Guide/pull/181#issuecomment-811985424

### CODE REVIEW

File: guide/payments/send.md [hunk 1]
```diff
@@ -145,18 +145,18 @@ If your application allows setting spending limits, and the current transaction
 
 {% include image.html
    image = "/assets/images/guide/payments/send/first-confirmation.svg"
-   alt-text = "Notifications after the transaction is broadcasted"
+   alt-text = "Notifications after the transaction is broadcast"
```
Comment (johnsBeharry) [inline guide/payments/send.md:148]: not sure about this. is it grammatically correct?

Comment (GBKS) [reply guide/payments/send.md:148]: Both appear to be acceptable. [See here](https://www.merriam-webster.com/words-at-play/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage). I personally prefer "broadcast", for whatever reason it sounds better, but YMMV.

File: guide/payments/send.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ The application can automatically estimate a fee and set it for the sender. This
 
 When allowing the sender to set fees, it is essential to communicate the estimated cost in their preferred currency and the estimated time until first confirmation. Be mindful that the way Bitcoin fees get calculated may not map to the sender's traditional financial experience. If you allow adjustment of fees, consider providing details on how it gets calculated.
 
-Human error with fee selection can lead to costly mistakes, and fee estimations are imperfect. Carefully consider if and how you expose transaction fees to senders.
+Human error with fee selection can lead to costly mistakes, and fee estimations are imperfect. Carefully consider if and how you expose transaction fees selection to senders.
```
Comment (GBKS) [inline guide/payments/send.md:109]:
```suggestion
Human error with fee selection can lead to costly mistakes, and fee estimations are imperfect. Carefully consider if and how you expose transaction fee selection to senders.
```

**Comment by pavlenex** [comment]
Closes #209

File: guide/payments/send.md (hunk 1 above)
Comment (johnsBeharry) [reply guide/payments/send.md:148]: Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo

**Review by johnsBeharry — APPROVED** [review]
Still not completely used to broadcast over broadcasted in some places, but if english works like this then I won't fight it.

Thanks for taking this on @paulosacramento!

**Pull request merged**